accounting-yaml/Xero_accounting_2.0.0_swagger.yaml

```diff
@@ -11587,6 +11591,248 @@ contact: {}, date:"2020-01-01", user:{} } ] } ] }'
           $ref: '#/components/responses/400Error'
       requestBody:
         $ref: '#/components/requestBodies/historyRecords'
+  '/Quotes/{QuoteID}/Attachments':
+    parameters:
+      - $ref: '#/components/parameters/requiredHeader'
+    get:
+      security:
+        - OAuth2: [accounting.attachments.read]
+      tags:
+        - Accounting
+      operationId: getQuoteAttachments
+      summary: Allows you to retrieve Attachments for Quotes
+      parameters:
+        - required: true
+          in: path
+          name: QuoteID
+          description: Unique identifier for Quote object
+          example: "00000000-0000-0000-000-000000000000"
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: Success - return response of type Attachments array of Attachment 
+          x-isAttachment: true
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Attachments'
+              example: '{
+                          "Id": "439c1573-3cd8-4697-a9f6-81fa651ee8f3",
+                          "Status": "OK",
+                          "ProviderName": "Xero API Partner",
+                          "DateTimeUTC": "\/Date(1550852630329)\/",
+                          "Attachments": [
+                            {
+                              "AttachmentID": "52a643be-cd5c-489f-9778-53a9fd337756",
+                              "FileName": "sample5.jpg",
+                              "Url": "https://api.xero.com/api.xro/2.0/Quotes/da962997-a8bd-4dff-9616-01cdc199283f/Attachments/sample5.jpg",
+                              "MimeType": "image/jpg",
+                              "ContentLength": 2878711
+                            }
+                          ]
+                        }'
+  '/Quotes/{QuotesID}/Attachments/{AttachmentID}':
```

Comment:
@SidneyAllen - We are trying to re-gen the sdk into xero-node.. getting the following error:
```
Errors: 
	-attribute paths.'/Quotes/{QuotesID}/Attachments/{AttachmentID}'. Declared path parameter QuotesID needs to be defined as a path parameter in path or operation level
```

Not 100% yet, but looks like `QuotesID` should actually be `QuoteID`?